draft-steele-cose-merkle-tree-proofs.md

```diff
@@ -338,11 +223,70 @@ inclusion-proof = #TBD_2([
 ])
 ~~~~
 
-#### Consistency Proof {#sec-rfc9162-sha256-consistency-proof}
+### Inclusion Proof Signature
+
+In a signed inclusion proof, the previous merkle tree root, maps to tree-size-1, and is a detached payload.
+
+Other specification's refer to signed inclusion proofs as "receipts",
```

Comment:
This is interesting. I've been struggling with the suggested need for signing inclusion proofs - in large scale native MT systems (like Ethereum) the proof stands alone (given a published root) and signatures are pointless. 

Are we saying here that actually 'signed inclusion proof' means 'a signed payload which includes a bunch of things including a stand-alone inclusion proof and some extra more semantically meaningful info about what the proof is for'? i.e. a complete SCITT sig?

Reply:
a signed inclusion proof is a signature over a merkle root, with the merkle audit path and otehr proof data in the unprotected header... such as that any party holding a leaf can recompute the root, and check the signature.

This allows parties to avoid storing or accessing a large shared data base, such as a blockchain.

Which is also valuable to blockchain based approaches that want to be trusted by parties that won't verify the chain.... for example, a vendor might issued signed inclusion proofs, and then other vendors can trust the transparency service vendor without running all their backend, etc...